src/main/java/multithreading/race/Main.java

```diff
@@ -0,0 +1,12 @@
+package multithreading.race;
```

Comment:
in this PR previous hw shouldn't be

Reply:
Got it. We spoke about merging these home works in the beginning.